include/boost/uuid/uuid.hpp

```diff
@@ -138,6 +138,16 @@ struct uuid
 
 public:
 
+    // Align uuid objects to 16 bytes on 64-bit platforms for performance.
+    // On most 32-bit platforms, default stack and dynamic memory alignment is lower than 16
+    // (typically 4 or 8), which makes it problematic to require uuid alignment of 16.
+#if (defined(__SIZEOF_POINTER__) && (__SIZEOF_POINTER__ >= 8)) || \
```

Comment:
How will this interact if the class is placed into another POD using packed alignment?

Reply:
Packed structs are not portable, and the behavior varies from one compiler to another. See [here](https://godbolt.org/z/49aEz1EGo). Basically, gcc and clang on System V ABI override `uuid` alignment, and MSVC respects `uuid` alignment and packs other data members with no explicit alignment requirements.